config/config.go

```diff
@@ -37,10 +38,21 @@ func InitConfig(project, region, bucket, table, awsProfile, owner, awsProviderVe
 
 // FindConfig loads a config and its version into memory
 func FindConfig(fs afero.Fs, configFile string) ([]byte, int, error) {
+	switch configFile {
```

Comment:
This seems like it will only read the json file if you explicitly give a `-c fogg.json`. We need to support using either yaml or json even when a config file is not specified at the command line.

Another way of thinking about this is that when we upgrade to this version, all commands should work exactly the same until you do the migration. So a repo with a json config file, when you run `fogg plan` should read the json file.